lib/flutter_datetime_formfield.dart

```diff
@@ -1,40 +1,68 @@
 library flutter_datetime_formfield;
 
+import 'dart:io' show Platform;
+
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:intl/intl.dart';
-import 'dart:io' show Platform;
 
 /// A date time pick form field widget.
 class DateTimeFormField extends StatelessWidget {
-  /// The initial date time, default value is 'DateTime.now()'.
+  /// The initial date time, default value is null.
   final DateTime initialValue;
+
   /// Save value function of form field.
   final FormFieldSetter<DateTime> onSaved;
+
   /// Validate function of form field.
   final FormFieldValidator<DateTime> validator;
+
   /// Whether validate every time, default value is false.
   final bool autovalidate;
   final bool enabled;
+
   /// The label of form field, default value is 'Date Time'.
   final String label;
+
   /// The format of displaying date time in form field, default value is 'DateFormat("EE, MMM d, yyyy h:mma")' in date and time mode,
   /// 'DateFormat("EEE, MMM d, yyyy")' in date only mode,
   /// 'DateFormat("h:mm a") in time only mode.
   final DateFormat formatter;
+
   /// Only show and edit date, default value is false.
   final bool onlyDate;
+
   /// Only show and edit time, default value is false. [onlyDate] and [onlyTime] cannot be set to true at the same time.
   final bool onlyTime;
+
   /// The first date time of picking, default value is 'DateTime(1970)'.
   final DateTime firstDate;
+
   /// The last date time of picking, default value is 'DateTime(2100)'.
   final DateTime lastDate;
 
+  /// Whether the iOS date picker should use 24-hour format or not, default is false
+  final bool use24hFormat;
+
+  /// If initialValue is null, the date time that is used to initialize the
+  /// date picker, default value is DateTime.now().
+  /// So if current value is not null, then selection interface will start with
+  /// current value, otherwise it will look at initialSelectionValue, and if
+  /// this is null, then it will start with DateTime.now()
+  final DateTime initialSelectionValue;
```

Comment:
What's the differences between initialSelectionValue and intialValue? You can use initialValue instead of adding a new parameter.